rosetta-ide/src/main/java/com/regnosys/rosetta/ide/quickfix/RosettaQuickFixProvider.java

```diff
@@ -80,4 +87,29 @@ public void fixDeprecatedMap(DiagnosticResolutionAcceptor acceptor) {
 			return List.of(edit);
 		});
 	}
+	
+	@QuickFix(RosettaIssueCodes.UNUSED_IMPORT)
+	@QuickFix(RosettaIssueCodes.DUPLICATE_IMPORT)
+	public void fixUnoptimizedImports(DiagnosticResolutionAcceptor acceptor) {
+		acceptor.accept("Optimize imports.", (Diagnostic diagnostic, EObject object, Document document) -> {
```

Comment:
I think you might be better off with another overload of this `accept` method. Have a look at the different options. :) (there is one on the level of text edits, which is what you are using, but perhaps it's easier to make a semantic edit to the model instead?